persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -71,6 +73,8 @@ class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor {
   val config = ConfigFactory.load()
   val flowParallelism = config.getInt("hmda.actor-flow-parallelism")
 
+  val submissionLarStats = context.actorOf(SubmissionLarStats.props(submissionId))
```

Comment:
Working on a fix, moving the declaration into `SubmissionManager` and passing references.  I'll have it up shortly.